about.html

```diff
@@ -0,0 +1,108 @@
+            <div class="header__title-container">
+                <h1 class="about-header__h1">Web Development Global Conference 2023</h1>
+            </div>
+            <hr class="about__header-line display-none">
+            <div class="about-header__text-box">
+                <p>Developers of the world, let get together! A space for sharing the joy of coding with peolple from over seventy different countries. Join us on September 12th, 2023 in Mexico City</p>
+            </div>
+            <p class="header__contact-us">Please contact us per Email for any further questions about Web Development Global Conference 2023</p>
+            <p class="header__mail">contact@thebestweb.com</p>
+        </div>
+    </header>
+    <main class="about">
+        <h2 class="subtitle-black">Web Development Conference Logo</h2>
+        <hr class="subtitle__line">
+        <p class="header__contact-us about__logo-description">The logo of the Web Development Conference was decided through the logo competition on September 8th 2015</p>
+        <div class="about__logo-container container">
+            <img src="./img/logo.png" alt="">
+        </div>
+    </main>
+    <section class="past-editions">
+        <h2 class="subtitle-black">See the past Web Conferences</h2>
+        <hr class="subtitle__line">
+        <p class="header__contact-us about__logo-description">Take a look to the last two Web Development Conferences wich took place in Bogotá and Madrid</p>
+        <div class="past-editions__card-container container">
+            <div class="past-editions__card1">
+                <p class="past-editions__year">2018</p>
+                <p class="past-editions__description">Web Development Conference in Bogotá</p>
+            </div>
+            <div class="past-editions__card2">
+                <p class="past-editions__year">2022</p>
+                <p class="past-editions__description">Web Development Conference in Madrid</p>
+            </div>
+        </div>
+    </section>
+    <section class="partners">
+        <h2 class="subtitle">Partner</h2>
+        <hr class="subtitle__line">
+        <div class="partners__logo-container">
+            <img class="footer__logo mozilla" src="./img/mozilla.png" alt="Google logo">
+            <img class="footer__logo"  src="./img/airbnb.png" alt="Airbnb logo">
+            <img class="footer__logo"  src="./img/google-logo.svg" alt="">
+            <img class="footer__logo"  src="./img/microsoft-logo.svg" alt="">
+            <img class="footer__logo"  src="./img/apple.png" alt="">
+        </div>
+    </section>
+    <footer class="license change-colors">
+        <img class="license__logo change-colors__logo" src="./img/cc-logo.svg" alt="creative commons logo">
+        <div>
+            <p class="license__text">2023 Creative Commons Mexico</p>
+            <p class="license__text">Some Rights Reserved</p>
+        </div>
+    </footer>
```

Comment:
- [ ] Your footer is great, but as the previous code reviewer mentioned, the colors of the footer on the mobile view of both pages should be white for the background and dark for the text and logo. Kindly invert the colors for the mobile view of the about page to match the template.
 - Your design:
 - 
<img width="439" alt="image" src="https://user-images.githubusercontent.com/61919617/217395564-ffc2ce08-d5b0-44ce-89f6-4b892684009e.png">

 - Template:
 - 
<img width="412" alt="image" src="https://user-images.githubusercontent.com/61919617/217395612-367f20a5-3be2-4072-bd76-4b7ee4443a6b.png">